src/content/getting-started.md

```diff
@@ -40,7 +40,9 @@ if required, however installing within the project allows for `yarn` script comm
 Next create a `Brocfile.js` file and an `src` folder for your source code.
 
 ```shell
-$ echo "module.exports = 'src'" > Brocfile.js && mkdir src
+$ echo "module.exports = 'src'" > Brocfile.js
```

Comment:
Should this be updated, considering the recent improvements re:

* esm
* function export